packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -15,9 +15,9 @@ type TBotBuilder = {
     app: AppStore;
     active_tab: number;
     has_started_onboarding_tour: boolean;
+    has_started_bot_builder_tour: boolean;
     is_preview_on_popup: boolean;
     setOnBoardTourRunState: (has_started_onboarding_tour: boolean) => boolean;
-    has_started_bot_builder_tour: boolean;
```

Comment:
@farabi-deriv it is better to keep the order, first the variables, then the functions
please check your files

Reply:
@maryia-matskevich-deriv I think this is fine, lets discuss these later with the team and decide what to follow because as far as Mehrzad explained to us previously, the order should be in ascending order alphabetically despite the type.